src/tasks/settings.html

```diff
@@ -0,0 +1,15 @@
+<h4>{{ 'task.settings.title' | translate }}</h4>
```

Comment:
Use `<legend>` element instead of '<h4>' as it looks better with the underline like in Send Email